care/emr/reports/renderer/template_engine.py

```
@@ -0,0 +1,212 @@
+    def _filter_currency(value: int | float | str, symbol: str = "₹") -> str:
+        if value is None or value == "":
+            return ""
+        try:
+            amount = float(value)
+        except (ValueError, TypeError):
+            return str(value)
+
+        negative = amount < 0
+        amount = abs(amount)
+        rupees = int(amount)
+        paise = int(round((amount - rupees) * 100))
```

Ruff (0.14.5)

Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)